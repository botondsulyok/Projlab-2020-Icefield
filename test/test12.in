field stable 0
field unstable 1 false 0
neighbours stable unstable
neighbours unstable stable
entity pb1 icebear stable
stats stable
stats unstable
stats pb1
move pb1 unstable
stats stable
stats unstable
stats pb1