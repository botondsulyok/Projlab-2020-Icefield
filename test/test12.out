id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: pb1
pullers: null
masters: null
neighbours: unstable

id: unstable
type: unstable
building: plains1
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: stable
capacity: 1
broken: false

id: pb1
type: icebear
iceUnit: stable
movebehaviour: amph0

id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: unstable

id: unstable
type: unstable
building: plains2
snow: 0
pickable: null
standers: pb1
pullers: null
masters: null
neighbours: stable
capacity: 1
broken: true

id: pb1
type: icebear
iceUnit: unstable
movebehaviour: amph0

